codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -190,6 +193,10 @@ protected EmbeddedChannel newContentCompressor(ChannelHandlerContext ctx, CharSe
         if (DEFLATE.contentEqualsIgnoreCase(contentEncoding) || X_DEFLATE.contentEqualsIgnoreCase(contentEncoding)) {
             return newCompressionChannel(ctx, ZlibWrapper.ZLIB);
         }
+        if (Brotli.isAvailable() && BR.contentEqualsIgnoreCase(contentEncoding)) {
+            return new EmbeddedChannel(ctx.channel().id(), ctx.channel().metadata().hasDisconnect(),
+                    ctx.channel().config(), new BrotliEncoder());
```

Comment:
Do we need another ctor overload that takes at least `quality` for `BrotliEncoder` in addition to `compressionLevel` and other params for gzip/deflate?